cluster/kube-down.sh

```diff
@@ -27,6 +27,7 @@ if [ -f "${KUBE_ROOT}/cluster/env.sh" ]; then
 fi
 
 source "${KUBE_ROOT}/cluster/kube-util.sh"
+source "${KUBE_ROOT}/cluster/common.sh"
```

Comment:
it's can not be changed if it has been designed like this, so i recover it, thank you!